docs/pages/dev/how_to/use_plugins/use_plugins.en.md

```diff
@@ -0,0 +1,369 @@
+               icon: 'plugin',
+               path: '/my-plugin',
+               permissions: ['plugin_access'],
+           },
+       ],
+       translations: {
+           en: {
+               'myPlugin.title': 'My Plugin',
+               'myPlugin.description': 'This is my plugin',
+           },
+           fr: {
+               'myPlugin.title': 'Mon Plugin',
+               'myPlugin.description': 'C\'est mon plugin',
+           },
+       },
+       baseUrls: {
+           api: '/api/my-plugin',
+       },
+       paramsConfig: {
+           'my-plugin': ['id', 'type'],
+       },
+   };
+   ```
+
+4. Add your plugin's main component:
+   ```typescript
+   // js/index.tsx
+   import React from 'react';
+
+   const MyPlugin: React.FC = () => {
+       return <div>My Plugin Content</div>;
+   };
+
+   export default MyPlugin;
+   ```
+
+5. Document your plugin:
+   ```markdown
+   # My Plugin
+
+   Description of what your plugin does.
+
+   ## Features
+   - Feature 1
+   - Feature 2
+
+   ## Required Permissions
+   - plugin_access
+
+   ## Configuration
+   Any specific configuration needed.
+   ```
+
+## Translations
+
+The plugin system includes comprehensive support for translations across the main application, plugins, and shared components. Here's how translations work:
+
+### Translation Structure
+
+Translations are organized in three main categories:
+1. **Main Application**: Core translations from the Iaso application
+2. **Plugin Translations**: Each plugin can provide its own translations
+3. **Component Translations**: Shared translations from bluesquare-components
+
+### Translation Files
+
+- Translation files should be JSON files named with their language code (e.g., `en.json`, `fr.json`)
+- Expected locations:
+  - Main app: `assets/js/apps/Iaso/domains/app/translations/{lang}.json`
+  - Plugins: `plugins/{pluginName}/js/src/constants/translations/{lang}.json`
+  - Components: `bluesquare-components/dist/locale/{lang}.json`
+
+### Build Process
+
+During the build process:
+1. Available languages are determined from the `AVAILABLE_LANGUAGES` environment variable (defaults to "en,fr")
+2. The system scans all translation directories for available languages
+3. Translations are combined into a single file for efficient loading
+4. Missing translations trigger warnings in the console
+5. The combined translations are exposed through Module Federation as `IasoModules/translations/configs`
+
+### Adding Translations to a Plugin
+
+To add translations to your plugin:
+1. Create a `js/src/constants/translations` directory in your plugin
+2. Add language files (e.g., `en.json`, `fr.json`)
+3. Structure your translations as a flat object with dot-notation keys:
+```json
+{
+    "myPlugin.title": "My Plugin Title",
+    "myPlugin.description": "Plugin description"
+}
+```
+
+4. Make sure to include at least an English translation file (`en.json`) as it will be used as a fallback for other languages
```

Comment:
This part should only explain the plugin specific part of the translations and link to the translation doc for more details